virttest/bootstrap.py

```diff
@@ -359,6 +359,8 @@ def _forced_or_detected(forced, detected):
             except Exception:
                 pass
         cpu_version = cpu.get_version() if hasattr(cpu, 'get_version') else None
+        # Replace special chars with _ to avoid bootstrap failure
+        cpu_version = re.sub(r'\W', '_', cpu_version) if cpu_version else cpu_version
```

Comment:
As the `cartesian_config` module defined, `-` is also a valid character, looks we can keep it.
https://github.com/avocado-framework/avocado-vt/blob/3769db6a6b4b3b7f14a3581cecd64e4e8dffd7b3/virttest/cartesian_config.py#L91

Reply:
Yes, I'll update.

Reply:
Updated.
```
cpu_version = 'v1.23-000*abc(xxx)_999'
cpu_version = re.sub(r'[^\w-]', '_', cpu_version) if cpu_version else cpu_version
cpu_version
'v1_23-000_abc_xxx__999'
```

Reply:
@luckyh Could you help review again? Thank you~